mlton/ssa/ssa-tree.sig

```diff
@@ -189,18 +177,32 @@ signature SSA_TREE =
                            prim: Type.t Prim.t,
                            return: Label.t} (* Must be nullary. *)
 
+            val size: t -> int
```

Comment:
Values in signature should be in alphabetical order; should go after `replaceVar`.